components/middleware/composer.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
What's the benefit of merging this with routing?

Reply:
Is middleware not a feature of routing? I don't see when or how someone would use Laravel middleware and not routing.

Also, the code examples for both were almost identical. Middleware just happened to include middleware. You could break out the middleware part into a separate PHP file within routing, but I would still keep it all contained in the Routing parent.

Reply:
I'm fine if it's separated within routing, but it was separated for a reason: the vast majority of people who use Laravel and want to use its routing elsewhere don't fully understand middleware. The goal of this project is to make people's learning curve and ease-of-porting-elsewhere as low as possible; that's why they were originally separate, if i remember correctly. So, separate file within routing is cool! But let's at least make it separate.